README.md

```diff
@@ -192,11 +192,23 @@ Install all the dependencies:
 
     $ yarn install
 
+#### The LTI frontend
 Run the build and copy the `iframe-resizer` host-side JS into your local `Django` assets:
 
-    $ yarn build
+    $ yarn build-lti
     $ yarn copy-iframe-resizer
 
+#### The "Standalone" frontend
+Run the build and copy the `iframe-resizer` host-side JS into your local `Django` assets:
+
+    $ yarn build-tests
+    $ yarn build-site
```

Comment:
You don't need to build an CRA to start it, you just need the lib used by the site to be build (eventually with a `yarn build-libs` command)